utilities/meta.yaml

```diff
@@ -22,6 +22,8 @@ requirements:
     - python
     - iris
     - lxml
+    - owslib
+    - requests
```

Comment:
`iris` installs `cartopy` that install` owslib`
`cartopy` installs `pyepsg` that install `requests`.

So I cannot see how the dependencies are missing.  Unless `iris` and cartopy are coming from a channel other than ours.